hack/scripts/generate_crds.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eo pipefail
+
+# This bash is the replacement for: ./boilerplate/_lib/container-make op-generate
+#                               or:  make -n generate
+
+# What you need to run this
```

Comment:
Or run it in the boilerplate backing image, which [contains](https://github.com/openshift/boilerplate/blob/753a9302de749bac0542e0cf4e8787f0ee389df3/config/build.sh) all of these things already.

Reply:
True, that's actually where I got all this stuff from.